File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q29.py
def main():
    #Input
    months = int(input("Enter a value in months: "))

    #Processing
    value_years = round(months / 24) 
    value_months = months % 24

    #Output
    print(f"The entered value of {months} months converted to years and months is equal to: {value_years} years and {value_months} months ")


main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q29.js
import { question } from "readline-sync";
function main(){
    //Input
    const months = parseInt(question("Enter a value in months: "))

    //Processing
    const value_years = Math.floor(months / 24)
    const value_months = months % 24

    //Output
    console.log(`The entered value in months of ${months} converted to years and months is equal to: ${value_years} years and ${value_months}`)

}
main()